com.playeveryware.eos/Runtime/Core/PlatformManager.cs

```diff
@@ -117,12 +119,54 @@ static PlatformManager()
             AddPlatformInfo(Platform.PS4,         "PS4",           "eos_ps4_config.json",     null, "PS4");
             AddPlatformInfo(Platform.PS5,         "PS5",           "eos_ps5_config.json",     null, "PS5");
             AddPlatformInfo(Platform.Switch,      "Switch",        "eos_switch_config.json",  null, "Switch");
-            //// TODO: Currently, there is no special config that is utilized for Windows - instead current implementation simply
-            //// relies on EpicOnlineServicesConfig.json, so for now this entry is different. What is commented below is what it *should* be to be consistent.
-            //// AddPlatformInfo(Platform.Windows,     "Windows",         "eos_windows_config.json", typeof(EOSWindowsConfig), ".dll");
-            //// For the time being, this is the entry for the Windows platform
 #endif
             AddPlatformInfo(Platform.Windows,     "Windows", "eos_windows_config.json", typeof(WindowsConfig), "Standalone");
+
+            // If external to unity, then we know that the current platform
+            // is Windows.
+#if EXTERNAL_TO_UNITY
+            CurrentPlatform = Platform.Windows;
+#else
+            // If the Unity Editor is currently running, then the "active"
+            // Platform is whatever the current build target is.
+#if UNITY_EDITOR
+            if (TryGetPlatform(EditorUserBuildSettings.activeBuildTarget, out Platform platform))
+#else
+            // If the Unity editor is _not_ currently running, then the "active"
+            // platform is whatever the runtime application says it is
+            if (TryGetPlatform(Application.platform, out Platform platform))
+#endif
+            {
+                CurrentPlatform = platform;
+            }
+            else
+            {
+                CurrentPlatform = Platform.Unknown;
+                Debug.LogWarning("Platform could not be determined.");
+            }
+#endif
+        }
+
+        public static PlatformConfig GetPlatformConfig()
+        {
+            PlatformConfig config = null;
+
+            // TODO: Confirm that usage of Reflection here is acceptable.
+            MethodInfo methodInfo = typeof(Config).GetMethod("Get");
+            MethodInfo genericMethod = methodInfo?.MakeGenericMethod(GetConfigType());
```

Comment:
I think that this can be changed by making `PlatformManager` partial and adding implementation to the restricted repository.

Reply:
In the very least, this should have it's value cached.

Reply:
Value is now cached, and a non-reflection dependent approach has been implemented. Marking conversation as resolved.